src/main/java/com/labsynch/labseer/api/ApiSaltController.java

```diff
@@ -283,7 +283,7 @@ public ResponseEntity<String> createFromJson(@RequestBody String json, @RequestP
 		}
 		if (validSalt & !dryrun) {
 			try {
-				salt = saltStructureService.saveStructure(salt);
+				salt = saltStructureService.saveStructureNoDupeCheck(salt);
```

Comment:
I think it would be preferrable to reuse the logic in saveStructure but just add a parameter "checkDuplicates" to the method.  This would involved making the other calls to `saveStructure` pass `true` to checkDuplicates but this is reasonable to do as I don't think there are many instances.

Reply:
Yea I was debating on it but was on the fence. Definitely can do that.

Reply:
For some reason thought there were more instances of the saveStructure utilized by SaltStructureService. My bad.